userinterface/vehicleparameterui.cpp

```diff
@@ -0,0 +1,124 @@
+/*
+ *     Copyright 2023 Rickard Häll   rickard.hall@ri.se
+ *     Published under GPLv3: https://www.gnu.org/licenses/gpl-3.0.html
+ */
+#include <iomanip>
+#include <sstream>
+#include "vehicleparameterui.h"
+#include "ui_vehicleparameterui.h"
+
+VehicleParameterUI::VehicleParameterUI(QWidget *parent) :
+    QDialog(parent),
+    ui(new Ui::VehicleParameterUI)
+{
+    ui->setupUi(this);
+}
+
+VehicleParameterUI::~VehicleParameterUI()
+{
+    delete ui;
+}
+
+void VehicleParameterUI::setCurrentVehicleConnection(const QSharedPointer<MavsdkVehicleConnection> &currentVehicleConnection)
```

Comment:
I would like to see this more general: only using VehicleConnection, not including any MAVSDK-specific headers.

Reply:
Fixed